### PR TITLE
minor fix: removed resetValidation and fix email regexp

### PR DESCRIPTION
**Pull request by kenta88**

### CODE REVIEW

File: src/utils/rules.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const regExp = {
     phoneNumbers: /^(?:[0-9\s\-\+\(\)])+$/, // eslint-disable-line no-useless-escape
     noOnlySpaces: /^\s*$/, // https://regex101.com/r/j4DA51/2/
-    email: /^[^@]+@[^@]+.[^@]+$/g, // https://regex101.com/r/dGDGRn/1/tests
+    email: /^[^@]+@[^@]+.[^@]+$/, // https://regex101.com/r/dGDGRn/1/tests
```
Comment (happylinks) [inline src/utils/rules.js]: https://regex101.com/r/RNreQI/1/tests

File: docs/main.js [hunk 2]
```diff
@@ -12415,9 +12413,11 @@ var _typeof2 = typeof Symbol === "function" && typeof Symbol.iterator === "symbo
             }, {
               key: 'setProperty',
               value: function setProperty(prop, value) {
-                return this.setModel(Object.assign({}, this.state.model, _defineProperty({}, prop, Object.assign({}, this.state.model[prop], {
+                var newModel = this.setModel(Object.assign({}, this.state.model, _defineProperty({}, prop, Object.assign({}, this.state.model[prop], {
```
Comment (happylinks) [inline docs/main.js]: where is the sourcecode for this?

Comment (kenta88) [inline docs/main.js]: this should not be here! sorry!

Comment (kenta88) [inline docs/main.js]: its seems coming from a previous fix.
https://github.com/Bynder/react-formulation/pull/19/files

I checked the pull request and the main.js it wasn't updated with the new build.

**Pull request merged**